missing `dotenv`

when trying missed-issues I'm getting this error 

```bash
$ npm install -g @mapbox/missed-issues 
$ npm link @mapbox/missed-issues 
$ missed-issues
module.js:471
    throw err;
    ^

Error: Cannot find module 'dotenv'

$ npm --version
3.10.10
$ node --version
v6.10.3
```
☝️ *updated with npm and node versions*

### Fix 
- add `dotenv` to package.json

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,8 @@
+# 1.2.0
+
+- Support for 👍 reactions[#3](https://github.com/mapbox/missed-issues/pull/3)
+- Fix bug where `dotenv` was not in the package.json
+
 # 1.1.0
 
 - Allow users to manage their configration via the cli [#2](https://github.com/mapbox/missed-issues/pull/2)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,4 +1,4 @@
-# 1.2.0
+# 1.2.0 (renamed to @mapbox/missed-issues)
 
 - Support for 👍 reactions[#3](https://github.com/mapbox/missed-issues/pull/3)
 - Fix bug where `dotenv` was not in the package.json

--- a/README.md
+++ b/README.md
@@ -2,8 +2,7 @@
 
 A CLI tool for finding issues that mention a team and no one from the team has responded to the issue.
 
-
-This will list all issues in the mapbox org that mention `@mapbox/teamname` where no member of that team has replied to the issue since the mention.
+This will list all issues in the mapbox org that mention `@mapbox/teamname` where no member of that team has replied to the issue since the mention. To help cut down on commenting for commenting's sake, it also accepts 👍 on the original post as acknowledgement of the issue if no comments below mention the team.
 
 ```sh
 > missed-issues --org mapbox --team teamname --token github_token
@@ -21,7 +20,17 @@ This will list all issues in the mapbox org that mention `@mapbox/teamname` wher
 - [ ] [#42 - Document the node module interface](https://github.com/mapbox/missed-issues/issues/new)
 ```
 
-## Flags
+## Usage
+
+`npm install @mapbox/missed-issues -g`
+
+Go to [Personal access tokens](https://github.com/settings/tokens) in GitHub and create a token with `read:org` and `repo` access and than run the follow command. This will store your token in a config file so that you don't have to type `--token [redacted]` every time you run `missed-issues`.
+
+```
+missed-issues config token [redacted]`
+```
+
+## Command Options
 
 **Required**
 

--- a/index.js
+++ b/index.js
@@ -41,9 +41,56 @@ module.exports = function(opts) {
         logger('Num Issues: ' + issues.length);
       })
     )
-    .then(() => fillInComments(opts, teamIssues)) //mutates teamIssues
-    .then(() => (teamIssues = filterIssues(teamIssues, teamMembers)));
+    .then(() => fillInComments(opts, teamIssues)) // mutates teamIssues
+    .then(() => (teamIssues = filterIssuesByComments(teamIssues, teamMembers)))
+    .then(() => fillInReactions(opts, teamIssues)) // mutates teamIssues
+    .then(
+      () =>
+        (teamIssues = filterIssuesByReactions(opts, teamIssues, teamMembers))
+    );
 };
+
+function filterIssuesByReactions(opts, issues, members) {
+  return issues.filter(iss => {
+    // only use +1 reactions
+    var hasMemberReaction = iss.reactions
+      .filter(rec => rec.content === '+1')
+      .some(rec => {
+        return members.indexOf(rec.user) > -1;
+      });
+
+    // we want to keep issues that members haven't reacted too
+    return hasMemberReaction === false;
+  });
+}
+
+function fillInReactions(opts, issues) {
+  return Promise.all(
+    issues.map(iss => {
+      var mentionedInComments = iss.comments.some(c => c.mentionsTeam);
+      if (mentionedInComments === false) return addReactions(opts, iss);
+      iss.reactions = []; // just so all issues look the same
+      return iss;
+    })
+  );
+}
+
+function addReactions(opts, iss) {
+  return safeGh(`repos/${opts.org}/${iss.repo}/issues/${iss.id}/reactions`, {
+    headers: {
+      accept: 'application/vnd.github.squirrel-girl-preview'
+    },
+    token: opts.token
+  }).then(resp => {
+    iss.reactions = resp.body.map(rec => {
+      return {
+        user: rec.user.login,
+        content: rec.content
+      };
+    });
+    return iss;
+  });
+}
 
 function logger(msg) {
   if (process.env.DEBUG === 'missed-issues') {
@@ -51,7 +98,7 @@ function logger(msg) {
   }
 }
 
-function filterIssues(issues, members) {
+function filterIssuesByComments(issues, members) {
   return issues.filter(t => {
     var lastMention = t.comments.reduce((m, c, i) => {
       if (c.mentionsTeam) return i;
@@ -99,7 +146,7 @@ function getComments(opts, ticket, page) {
       };
     });
     if (hasNext) {
-      return getComments(opts, ticket, page, comments.length).then(c =>
+      return getComments(opts, ticket, page + 1, comments.length).then(c =>
         comments.concat(c)
       );
     }

--- a/package.json
+++ b/package.json
@@ -36,6 +36,7 @@
     "*.js": ["eslint .", "prettier --single-quote --write", "git add"]
   },
   "dependencies": {
-    "dotenv": "^4.0.0"
+    "dotenv": "^4.0.0",
+    "gh-got": "^6.0.0"
   }
 }

--- a/package.json
+++ b/package.json
@@ -35,5 +35,7 @@
     "*.json": ["prettier --single-quote --write", "git add"],
     "*.js": ["eslint .", "prettier --single-quote --write", "git add"]
   },
-  "dependencies": {}
+  "dependencies": {
+    "dotenv": "^4.0.0"
+  }
 }

